### ISSUE TITLE
How can I use multi instances?

### ISSUE THREAD

**Issue by quydo** [prasmussen/gdrive#233]
Dear team,

I have a Linux machine, how can I use multi instances gdrive to sync one folder with multi Google Drive account?

**Comment by Drezzler** [comment]
I dont think you can sync with multiple accounts simultaneously. But you can sync them in a sequence , using each account separately.
